state/migration_export.go

```diff
@@ -803,6 +803,12 @@ func (e *exporter) addApplication(ctx addApplicationContext) error {
 		workloadVersionKey := unit.globalWorkloadVersionKey()
 		exUnit.SetWorkloadVersionHistory(e.statusHistoryArgs(workloadVersionKey))
 
+		globalCCKey := unit.globalCloudContainerKey()
```

Comment:
We were going to delete this right?
It needs to be done when juju/description is updated